jax/numpy/lax_numpy.py

```diff
@@ -2708,11 +2708,23 @@ def polyder(a, m=1):
   coeff = (arange(len(a), m, -1) - 1 - arange(m)[:, newaxis]).prod(0)
   return a[:-m] * coeff
 
-def _trim_zeros(a):
-  for i, v in enumerate(a):
-    if v != 0:
-      return a[i:]
-  return a[:0]
+
+def _trim_zeros(filt, trim='fb'):
+  first = 0
+  if 'f' in trim:
+    for i in filt:
+      if i != 0:
+        break
+      first += 1
+
+  last = len(filt)
+  if 'b' in trim:
+    for i in filt[::-1]:
+      if i !=0:
+        break
+      last -= 1
```

Comment:
It seems like it would be cleaner to continue using `enumerate` here, maybe something like this?
```python
def trim_zeros(filt, trim='fb'):
  start = 0
  end = 0
  if 'f' in trim.lower():
    for start, v in enumerate(filt):
      if v != 0:
        break
  if 'b' in trim.lower():
    for end, v in enumerate(filt[::-1]):
      if v != 0:
        break
  return filt[start: len(filt) - end]
```